lib/services/dgg-rolling-chat-cache.js

```diff
@@ -43,6 +57,7 @@ class ChatCache {
   addMessageToCache(user, message) {
     if (message.length >= this.viewerMessageMinimumLength) {
       this.addMessageToViewerMap(user, message);
+      this.addMessageToViewerUrlMap(user, message);
```

Comment:
short links may not be detected.

The message cache was mostly made for a specific spam filter. It does similarity detection of any two messages by a user. So to only check the ones that aren't like

hey bob
hey jim

it doesn't cache messages below a certain size.

Small links wouldn't get detected. So if you're filtering on links, I'd recommend adding any message to the viewer urlmap regardless of length, because you've got the URL filter.